xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -426,6 +498,114 @@ private String processRouteConfig(RouteConfiguration config) {
     return null;
   }
 
+  /**
+   * Handles EDS response, which contains a list of ClusterLoadAssignment messages. Each
+   * ClusterLoadAssignment message contains information for endpoints in a cluster, grouped
+   * by localities. The response is NACKed if contains invalid information for gRPC's usage.
```

Comment:
"Each ClusterLoadAssignment message contains information for endpoints in a cluster, grouped by localities" not need to specify here.

Reply:
Rephrased.